playground/scripts/template/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is it possible to include defaults here that make the example installable standalone?

Reply:
The Lit libraries are intentionally left out of the `package.json` so that examples will automatically default to using the versions from the current branch of the monorepo.

I wasn't thinking of standalone examples as a P0/P1 use case, but if we want to support that, I think the best way of doing it would be to add an `export:standalone` script that adds deps into the exported `package.json`.